### PR TITLE
add my information

### PR DESCRIPTION
**Pull request by Y0505**

### CODE REVIEW

**Comment by Y0505** [comment]
I try to add my infomation to README.md and use PR

File: README.md [hunk 1]
```diff
@@ -4,4 +4,5 @@ The list below is the names of students and participants in my professional cour
 
 | First Name      | Last Name | College | Courses | Social Media | Comments |
 | :----: | :----: | :----: | :----: | :----: | :----: |
-| Example     | Example     | - | PHP, HTML, CSS | [Telegram](https://t.me) | My dude |
```
Comment (azadsoul) [inline README.md:7]: You should not delete the example!

File: README.md [hunk 2]
```diff
@@ -4,4 +4,5 @@ The list below is the names of students and participants in my professional cour
...
+| Yaser     | Montazeri     | MFD | JavaScript, HTML, CSS | - | My first face-to-face course |
```
Comment (azadsoul) [inline README.md:7]: Your college correct abbr is 'MFT'

**Review by azadsoul — COMMENTED** [review]
Please fix the issues and PR again

**Pull request closed without merging**